src/components/strategies/marketPriceIndication/useMarketIndication.ts

```diff
@@ -42,24 +42,26 @@ export const useMarketIndication = ({
     }
   }, [order.isRange, order.price, order.max, order.min, buy, marketPrice]);
 
-  const marketPricePercentage = useMemo(() => {
-    const getMarketPricePercentage = (price: string) => {
-      const value = new SafeDecimal(price || 0);
-      if (value.eq(0)) return value;
-      return new SafeDecimal(value)
-        .minus(marketPrice)
-        .div(marketPrice)
-        .times(100);
-    };
-    return {
-      min: getMarketPricePercentage(order.min),
-      max: getMarketPricePercentage(order.max),
-      price: getMarketPricePercentage(order.price),
-    };
-  }, [order.max, order.min, order.price, marketPrice]);
+  const marketPricePercentage = useMemo(
+    () => ({
+      min: marketPricePercent(order.min, marketPrice),
+      max: marketPricePercent(order.max, marketPrice),
+      price: marketPricePercent(order.price, marketPrice),
+    }),
+    [order.max, order.min, order.price, marketPrice]
+  );
 
   return {
     isOrderAboveOrBelowMarketPrice,
     marketPricePercentage,
   };
 };
+
+export const marketPricePercent = (
```

Comment:
I think that if marketPrice is 0, instead of dividing by zero, this function should return 0. Same for `isOrderAboveOrBelowMarketPrice` above returning false for marketPrice === 0. 

To prevent showing warnings when there is no marketPrice either external or from the user, like in disposable and recurring strategies